.factory/automation.yml

```diff
@@ -104,57 +104,42 @@ build:
       dependencies: [build]
       command: |
         bazel run @typedb_dependencies//tool/bazelinstall:remote_cache_setup.sh
-        bazel test //tests/behaviour/connection/... --test_output=streamed
+        bazel test //tests/behaviour/connection:test_connection --test_output=streamed  --test_arg="--test-threads=1"
```

Comment:
We might actually be able to parallelise if the features use different databases.